6.x/base-widgets.md

```diff
@@ -367,7 +367,10 @@ Allows you to include multiple widgets in the div attributes of your choice. For
 ]
 ```
 
-Anything you specify on this widget, other than ```type``` and ```content```, has to be a string, and will be considered an attribute of the "div" element.
+Anything you specify on this widget, other than ```type``` and ```content```, has to be a string, and will be considered an attribute of the "div" element. 
+For example, in the above snippet, the "div" element will end up with a ```class``` attribute with a value of ```row```, which is a string. 
+This value has to be a string of one of more class names, for example ```row border border-4 border-danger```.
```

Comment:
```suggestion
For example, in the above snippet, the "div" element will end up with a ```class``` attribute with a value of ```row```, which is a string. 
This value has to be a string of one of more class names, for example ```row border border-4 border-danger```.
```

Maybe instead "verbosely" explaining this and having the need for two examples we:
- use two classes, `'class' => 'row my-custom-widget-class',`
- add one more attribute, `'custom-attribute' => 'my-custom-value'`
- show the generated output `<div custom-attribute="my-custom-value" class="row my-custom-widget-class">`

Do you think it will be better like I suggested above ?

Reply:
Yes, that is much better. I had thought of adding the following to the widget:

'style' => 'border: solid 3px  red'

But your three suggestions together should be much clearer.